katz/src/main/kotlin/katz/effects/Duration.kt

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
We should have attribution in files like License or where they belong like we do in the typelevel cats case.